package.json

```diff
@@ -26,7 +26,11 @@
     "prepublishOnly": "git clean -xdf"
   },
   "dependencies": {
+    "blessed": "^0.1.81",
```

Comment:
i didn't see any usage of this library

Reply:
With this and `multiview` I was experimenting with ways to run parcel and npm in parallel and show the output from both to the user. I got it working, but after trying it I decided that the output of `npm` wasn't interesting enough to be worth displaying.